solution/ui/regulations/css/scss/partials/_search.scss

```diff
@@ -5,164 +5,149 @@
 
-    .search-options {
-        margin: $spacer-2 0 $spacer-3;
+                    a {
+                        text-decoration: none;
+                    }
+                }
 
-        a {
-            text-decoration: none;
-            margin-right: $spacer-2;
-            padding: 0 0.9em 0.3em 0;
+                &__snippet {
+                    font-size: $font-size-sm;
+                }
 
-            &.selected {
-                border-bottom: 2px solid $primary_link_color;
+                .search-highlight {
+                    font-weight: bold;
+                }
             }
-        }
 
-        .fa-search {
-            margin-right: 0.3em;
-        }
+            .reg-results-content {
+                @include common-results-styles;
 
-        .fa-angle-down {
-            margin-left: 0.3em;
-        }
+                .result {
+                    &__context {
+                        margin-bottom: 4px;
+                    }
 
-        label {
-            color: $primary_link_color;
-            cursor: pointer;
-
-            &:hover {
-                color: $primary_link_hover;
+                    &__link {
+                        margin-bottom: 4px;
+                    }
+                }
             }
-        }
-    }
-
-    .hierarchy {
-        float: right;
-    }
-
-    // Results
 
-    .result {
-        margin: $spacer-3 0;
-        line-height: $font-size-2xl;
-
-        .results-part {
-            font-size: $font-size-xs;
-        }
-
-        .results-section {
-            font-size: $font-size-md;
-
-            a {
-                text-decoration: none;
+            .resources-results-content {
+                @include common-results-styles;
+
+                .related-sections {
+                    margin-top: 8px;
+                    margin-bottom: 0px;
+
+                    .related-sections-title {
+                        @include common-context-related-sections;
+                        text-transform: uppercase;
+                    }
+
+                    .related-section-link {
+                        @include common-context-related-sections;
+                        a {
+                            text-decoration: none;
+                        }
+                    }
+                }
             }
-        }
 
-        .results-preview {
-            font-size: $font-size-sm;
-        }
+            .search-results-count {
+                font-weight: bold;
+                margin: $spacer-3 0;
 
-        .search-highlight {
-            font-weight: bold;
+                h2 {
+                    border-bottom: 2px solid $mid_blue;
+                    margin-block-end: 1em !important;
+                }
+            }
         }
-    }
 
-    // Right Sidebar
+        .pagination-expand-row {
+            display: flex;
+            flex-direction: row;
+            justify-content: center;
+            margin-bottom: 100px;
```

Comment:
some styles, like this one, aren't new -- they have been removed from the style tag of a Vue component and moved into a SCSS stylesheet.  These styles were moved from Search.vue to here.